src/components/CoinOverview/CoinOverview.js

```diff
@@ -57,50 +57,50 @@ class CoinOverview extends React.Component {
               </tr>
             </StyledTHead>
             <tbody className="space-x-4">
-              {list.map((obj, index) => (
+              {list.map((coin, index) => (
                 <tr>
```

Comment:
you need to add key 

https://reactjs.org/docs/lists-and-keys.html#basic-list-component